content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -485,6 +485,39 @@ fn inspect_changes_system<T: Component + Debug>(q: Query<Ref<T>>) {
 
 We are also deprecating `ChangeTrackers<T>`, which is the old way of inspecting a component's change ticks. This type will be removed in the next version of Bevy.
 
+## Renderer Optimizations
+
+<div class="release-feature-authors">authors: @danchia, @superdump, james7132, @kurtkuehnert</div>
```

Comment:
```suggestion
<div class="release-feature-authors">authors: @danchia, Rob Swain (@superdump), james7132, @kurtkuehnert</div>
```

Reply:
Although I'm not sure I directly contributed to any of the mentioned PRs. If not then I shouldn't be mentioned at all here. :)

Reply:
You authored https://github.com/bevyengine/bevy/pull/7069, so I think your name should be mentioned here.